client/src/main/python/slipstream/utils/tasksrunner.py

```diff
@@ -94,12 +95,14 @@ def work(self):
 
 class ThreadWrapper(Thread):
     def __init__(self, group=None, target=None, name=None,
-                 args=(), kwargs=None, verbose=None, exc_queue=None):
+                 args=(), kwargs=None, verbose=None, exc_queue=None,
+                 daemonic=True):
         if verbose != None and int(verbose) >= 3:
             verbose = True
         super(ThreadWrapper, self).__init__(group=group, target=target,
                                             name=name, args=args,
                                             kwargs=kwargs, verbose=verbose)
+        self.setDaemon(daemonic)
```

Comment:
`setDeamon()` simply wraps `daemon` decorator based setter and is in 2.7 as well.  But I'm OK for changing it to more pythonic way of the code.